db/schema.rb

```diff
@@ -40,5 +40,22 @@
     t.index ["name"], name: "index_publications_on_name"
   end
 
+  create_table "users", force: :cascade do |t|
+    t.string "email", default: "", null: false
+    t.string "encrypted_password", default: "", null: false
+    t.string "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.integer "sign_in_count", default: 0, null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.inet "current_sign_in_ip"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.